src/jdk.dynalink/share/classes/jdk/dynalink/linker/GuardingDynamicLinkerExporter.java

```diff
@@ -44,24 +44,10 @@
  * @since 9
  */
 public abstract class GuardingDynamicLinkerExporter implements Supplier<List<GuardingDynamicLinker>> {
-    /**
-     * The name of the runtime permission for creating instances of this class.
-     * @apiNote
-     * This permission cannot be used for controlling access to resources
-     * anymore as the Security Manager is no longer supported.
-     */
-    public static final String AUTOLOAD_PERMISSION_NAME = "dynalink.exportLinkersAutomatically";
```

Comment:
This at least is not used externally, it seems so I guess we can remove it no strings attached.